### PR TITLE
Remove simple history from models

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
LEARNER-3332

### CODE REVIEW

File: ecommerce/extensions/payment/migrations/0019_auto_20171122_1220.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('payment', '0018_create_stripe_switch'),
+    ]
+
+    operations = [
```
Comment (clintonb) [inline ecommerce/extensions/payment/migrations/0019_auto_20171122_1220.py:15]: What's going on here? Why is this model being changed?

**Review by clintonb — CHANGES_REQUESTED** [review]
There are seemingly unrelated migrations. Also, your tests are failing.

**Comment by clintonb** [comment]
FYI @edx/learner @edx/devops

**Comment by feanil** [comment]
Won't these removals try to remove tables that are really large?  Also, won't this break the deploy in multiple ways?  Is it possible to stop using the simplehistory module without generating these migrations.  I'd rather we do this cleanup by hand after the software no longer cares about these tables.  We should also make a note about this on the Hawthorn Page https://openedx.atlassian.net/wiki/spaces/COMM/pages/158045104/Hawthorn

**Pull request closed without merging**